examples/benchmark/README.md

```diff
@@ -0,0 +1,163 @@
+# Benchmarking S3 Connector for PyTorch
```

Comment:
There a bunch of things that can be improved from a grammar point of view. Can we please run a grammar check on this?

Reply:
Hah! because I wrote part of it :) I will start taking English classes.